app/src/main/res/layout/fragment_owner_details.xml

```diff
@@ -0,0 +1,176 @@
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/owner_name"
+            style="@style/TextDark.Bold"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="18dp"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:drawablePadding="11dp"
+            android:drawableRight="@drawable/ic_edit_green_24dp"
+            app:layout_constraintBottom_toTopOf="@id/owner_address"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/owner_blockie"
+            tools:text="Key 1" />
+
+        <TextView
+            android:id="@+id/owner_address"
+            style="@style/TextMedium"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="72dp"
+            android:layout_marginStart="72dp"
+            android:layout_marginTop="10dp"
+            android:background="@drawable/background_selectable"
+            android:ellipsize="end"
+            android:gravity="center"
+            android:lines="3"
+            android:textColor="@color/text_emphasis_low"
+            android:textSize="16sp"
+            app:layout_constraintBottom_toTopOf="@id/owner_key_address_label"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/owner_name"
+            tools:text="0x9bebe3b9e7a461e35775ec935336891edf856da2" />
+
+        <ImageView
+            android:id="@+id/link"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginStart="12dp"
+            android:background="@drawable/background_selectable"
+            android:src="@drawable/ic_link_green_24dp"
+            app:layout_constraintBottom_toBottomOf="@id/owner_address"
+            app:layout_constraintStart_toEndOf="@id/owner_address"
+            app:layout_constraintTop_toTopOf="@id/owner_address" />
+
+        <TextView
+            android:id="@+id/owner_key_address_label"
+            style="@style/TextDark.Bold"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:text="@string/signing_owner_details_key_address"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/owner_address" />
+
+        <com.google.android.material.imageview.ShapeableImageView
+            android:id="@+id/owner_qr_code"
+            android:layout_width="124dp"
```

Comment:
This looks rather small on my phone. But it is probably the same size as in the share dialog?

Reply:
yes

Reply:
yes